playbooks/lets-encrypt.yml

```diff
@@ -0,0 +1,65 @@
+---
+# lets-encrypt.yml asks questions about Streisand instance domain name, which
+# is used to request TLS certificates from Let's Encrypt. If user fails to
+# answer these questions, lets-encrypt role would not be run, and self-signed
+# certificates are generated and used instead.
+- name: Collect information about the Streisand domain
+# =========================================
+  hosts: streisand-host
+  gather_facts: no
+
+  vars_files:
+    - roles/lets-encrypt/vars/main.yml
+
+  vars_prompt:
+    - name: "streisand_domain_var"
+      prompt: |
+        Which domain do you want to use to host Streisand server?
+
+        This is an optional question. If you have a domain that points to your
+        Streisand server, the installation scripts can request Let's Encrypt
+        HTTPS certificates for you automatically.  If you do not provide one or
+        the request fails, self-signed certificates will be used instead.
+
+        If you have just created a new cloud server in previous steps, it is
+        good time to point your domain to your server's public address, as DNS
+        changes take time to propagate.
+
+        In order to use Let's Encrypt, you must agree to its agreements at
+        {{ le_agreements }}.
```

Comment:
This templating won't work in a `prompt` and ends up displaying the literal `{{ le_agreement }}`.

Reply:
@cpu Ah the fact that `vars_prompt` does not support templating is just as annoying as the fact that `vars_prompt` does not support `when` statement: https://github.com/ansible/ansible/issues/13263

I have removed this line.